src/edit/administrativeNoteFields.js

```diff
@@ -11,10 +11,16 @@ import {
   TextArea,
 } from '@folio/stripes/components';
 
+import {
+  validateFieldLength
+} from '../utils';
+
 const AdministrativeNoteFields = () => {
   const { formatMessage } = useIntl();
 
   const administrativeNoteLabel = formatMessage({ id: 'ui-inventory.administrativeNote' });
+  const START_WITH_MAX_LENGTH = 32000;
+  const validateStartWithMaxLength = value => validateFieldLength(value, START_WITH_MAX_LENGTH);
```

Comment:
The name `validateStartWithMaxLength` is confusing, let's do this:
```
validate={value => validateFieldLength(value, START_WITH_MAX_LENGTH)}
```

Reply:
ok - good point